support/build/Makefile.rules

```diff
@@ -290,24 +289,21 @@ tmp_depfile = $(dir $1).$(notdir $1).d
 depflags = -Wp$(comma)-MD$(comma)$(call tmp_depfile,$(1))
 define build_cmd_fixdep =
 	$(call build_cmd,$1,$2,$3,$4)
-	$Q $(UK_FIXDEP) $(call tmp_depfile,$3) $3 '$(call strip,$4)' \
+	$Q $(UK_FIXDEP) $(call tmp_depfile,$3) $3 'command' \
```

Comment:
This is the relevant line for the fixdep issue. `$(call strip,$4)` can be to large for a single argument, but for correctness the argument is needs to change whenever `$(4)` changes.